OpenRA.Mods.Cnc/Activities/Teleport.cs

```diff
@@ -65,7 +65,14 @@ public override Activity Tick(Actor self)
 
 			self.Trait<IPositionable>().SetPosition(self, destination);
 			self.Generation++;
-
+            var positionable = self.Trait<IPositionable>();
+            positionable.SetPosition(self, destination);
```

Comment:
This duplicates the lines above?
Imho it'd be good to reduce the whole block (lines 66 to 75) to
```
var positionable = self.Trait<IPositionable>();
positionable.SetPosition(self, destination);

var mobile = positionable as Mobile;
if (mobile != null)
    mobile.EnteringCell(self);

self.Generation++;
```

(Note that `positionable as Mobile` returns `null` if `positionable` is not an instance of `Mobile`. This way we don't need to check twice [only one `as` instead of an `is` and an `as`]).

Also please uses tabs instead of spaces for indentation.